examples/example_struct/make_bindings_via_cmake.py

```diff
@@ -48,8 +53,8 @@ def make_bindings_code(all_includes_fn):
                + ('--config config.cfg ' if use_pybind_stl else '') +
                f' {all_includes_fn} -- -std=c++11 '
                f'-I{this_project_include} -DNDEBUG -v').split()
-    print(' '.join(command))
-    ret = subprocess.call(command)
+    print('BINDER COMMAND:', ' '.join(command))
+    subprocess.call(command)
```

Comment:
Here and in other places of this script: it probably a good idea to replace `subprocess.call` with `subprocess.check_call` so errors during call terminate the script (which is what we want most of the time). (see https://docs.python.org/3/library/subprocess.html#subprocess.check_call for details)

Reply:
```suggestion
    subprocess.check_call(command)
```